prime-router/src/main/kotlin/Translator.kt

```diff
@@ -29,29 +29,7 @@ class Translator(private val metadata: Metadata) {
         return metadata.organizationServices.map { service -> translateByService(input, service, defaultValues) }
     }
```

Comment:
@mauricereeves , I'm having trouble seeing what changes were made to Translator.kt - could you just put in a brief comment?

At first, it looked like major changes, but the more I looked at it, it seems like some methods just got moved around a bit.

Thanks.

Reply:
I was originally going to have the synthetic data happen in Translator, but it was increasingly complex and problematic so I rolled back my changes. There shouldn't be any real changes to review there.

Reply:
gracias